docs/event-mesh/03-01-event-processing.md

```diff
@@ -11,6 +11,10 @@ This diagram explains the event flow in Kyma, from the moment the Application se
 
     >**NOTE:** The HTTP Source Adapter accepts only CloudEvents in version 1.0.
 
+  Before Kyma Version 1.11, the application sent events to the `/v1/events` endpoint exposed by the [Event Service](https://github.com/kyma-project/kyma/blob/master/components/event-service/README.md). This service was responsible for sending events to the Event Bus, which back then handled event processing and delivery. In Event Mesh, the application sends events to the directly exposed `/events` endpoint. These events must comply with the [CloudEvents specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md).
+
+  It may happen, that you still use the previous endpoint to receive the events. That is why the Event Service's logic was refactored into a compatibility layer to ensure that the events received at the `/v1/events` endpoint can be forwarded to the subscriber. To achieve that, it consumes the events in the non-CloudEvent format, transforms them into CloudEvents, and propagates to the new Event Mesh. 
```

Comment:
```suggestion
  It may happen that you still use the previous endpoint to receive events. That is why we equipped the Event Service with an additional compatibility layer that ensures the events sent to the `/v1/events` endpoint can be forwarded to the subscriber. To achieve that, the compatibility layer consumes the events in the non-CloudEvent format, transforms them into CloudEvents, and propagates them to the new Event Mesh. 
```